addons/sale/wizard/sale_order_discount_views.xml

```diff
@@ -21,6 +21,14 @@
                                 <field name="discount_percentage"
                                        invisible="discount_type not in ('so_discount', 'sol_discount')"
                                        widget="percentage" nolabel="1"/>
+                                <field
+                                    name="tax_ids"
+                                    widget="many2many_tags"
+                                    options="{'no_create': True}"
+                                    context="{'search_view_ref': 'account.account_tax_view_search'}"
```

Comment:
Is it different from the default view for taxes? If not, no need to specify it here.